geth/common/notification.go

```diff
@@ -0,0 +1,16 @@
+package common
+
+import "github.com/NaySoftware/go-fcm"
+
+// Notification manages Push Notifications and send messages
+type Notification interface {
+	Notify(token string) string
+	Send() error
+}
+
+// Messaging manages send/notification messaging clients
+type Messaging interface {
+	NewFcmRegIdsMsg(list []string, body interface{}) *fcm.FcmClient
```

Comment:
Can we rename this to `NewClient` or `NewFCMClient`? 

`NewFCMReqIdsMsg` does not really fit it's purpose and seem more of made up due to method's argument. I think naming methods with regards to purpose and return value is more idiomatic. :)

Reply:
This is copy of methods of "github.com/NaySoftware/go-fcm" to create interface on our side. So i need exact match of methods and cant change `NewFCMReqIdsMsg` interface.

Reply:
Alright.